cmd/samples/README.md

```diff
@@ -10,9 +10,18 @@ See instructions for running the Cadence Server: https://github.com/uber/cadence
 
 ## Steps to run samples
 ### Build Samples
+* Build all the workflows at once
 ```
 make
 ```
+* Build a workflow individually
+```
+make <WORKFLOW NAME>
+
+Example:
+    make delaystart 
```

Comment:
nit: I'd avoid adding an extra example here. `make helloworld` seems sufficient to show how to build an individual workflow